sulu/article-bundle/2.0/config/packages/sulu_article.yaml

```diff
@@ -0,0 +1,16 @@
+sulu_article:
+    index_name: '%env(resolve:ELASTICSEARCH_INDEX)%'
+    hosts:
+        - '%env(resolve:ELASTICSEARCH_HOST)%'
+
+sulu_route:
+    mappings:
+        Sulu\Bundle\ArticleBundle\Document\ArticleDocument:
+            generator: schema
+            options:
+                route_schema: '/articles/{object.getTitle()}'
+        Sulu\Bundle\ArticleBundle\Document\ArticlePageDocument:
+            generator: article_page
+            options:
+                route_schema: '{translator.trans("page")}-{object.getPageNumber()}'
+                parent: '{object.getParent().getRoutePath()}'
```

Comment:
here should also be the path config for ongr:

see
https://github.com/sulu/SuluArticleBundle/blob/master/Resources/doc/installation.md

Reply:
> FYI: A recipe should not modify other bundles config.

Probably we have to create an own recipe for `handcraftedinthealps/elasticsearch-bundle`. Or better prepending ongr config in the `ArticleExtension`. WDYT?

Reply:
No it should just inside the `sulu_article.yaml` as it is ongr config for the sulu article bundle.

Reply:
Can we add the `ongr_elasticsearch:` configuration atleast as commented out in this file so the dev sees what he need to activate for the bundle to work.